src/pages/Checkout/CheckoutPage.vue

```diff
@@ -879,6 +878,7 @@ export default {
 		},
 		verificationComplete() {
 			this.verificationSubmitted = true;
+			this.handleTeamForm();
```

Comment:
This is probably a fine location to handle the "after" the verification process. However, we also need to check trigger the check for Team Join on page load IF verification isn't needed.

I suggest you add an else statement around line 865 in the `getPromoInformatinoFromBasket` method

Reply:
An else statement was added to trigger the **handleTeamForm** method